bin.js

```diff
@@ -45,13 +50,21 @@ function getConfig(file) {
     // point to the particular config
     configFile: file,
   })
-  var config = cliEngine.getConfigForFile(file)
-  return config
+  return cliEngine.getConfigForFile(file)
 }
 
-function getRules() {
-  var configFile = getConfigFile()
-  var config = getConfig(configFile)
+function getCurrentRules() {
```

Comment:
My bad, i should have removed the arguments from the function call.
If I have to add it as an argument I need to rename the variable, otherwise eslint complains, could you confirm which is better?